public/data/_list.json

```diff
@@ -0,0 +1,29 @@
+[
+  { "username": "eddiejaoude", "avatar": "https://github.com/eddiejaoude.png" },
+  {
+    "username": "nhcarrigan",
+    "avatar": "https://avatars.githubusercontent.com/u/63889819?v=4"
+  },
+  { "username": "stemount", "avatar": "https://github.com/stemount.png" },
+  {
+    "username": "iasonathanasiosgiatsios",
+    "avatar": "https://github.com/iason9.png"
+  },
+  { "username": "FayasNoushad", "avatar": "https://github.com/FayasNoushad.png" },
+  { "username": "mohammad", "avatar": "https://github.com/mohammad4kh.png" },
+  {
+    "username": "verma-kunal", 
+    "avatar": "https://github.com/verma-kunal.png"
+  },
+  {
+    "username": "Skyhero-admin", 
+    "avatar": "https://github.com/Skyhero-admin.png"
+  },
+  { "username": "krishdevdb", "avatar": "https://github.com/krishdevdb.png" 
+  },
+  { "username": "vinzvinci", "avatar": "https://github.com/vinzvinci.png" },
+  { "username": "voidz", "avatar": "https://github.com/voidz7.png" },
+  { "username": "suhail34", "avatar": "https://github.com/suhail34.png" },
+  { "username": "Rajiv Tiwari", "avatar": "https://github.com/rajivransom.png" },
+  { "username": "RoshanMhatre", "avatar": "https://github.com/RoshanMhatre.png" }
+]
```

Comment:
This file not needed, You can delete this.

Reply:
Should I delete only mine or others too?